.github/workflows/tpip-check.yml

```diff
@@ -48,3 +47,30 @@ jobs:
       - name: Check Licenses
         run: go-licenses check . --ignore github.com/Open-CMSIS-Pack/cbuild --disallowed_types=forbidden,restricted
         working-directory: ./cmd/cbuild
+
+  commit-changes:
+    needs: [ check-licenses ]
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.GH_ADMIN_TOKEN }}

```

Comment:
@JonatanAntoni I request you add a secret named `GH_ADMIN_TOKEN` (having `git push` permission to the branch) for `cbuild` and `cpackget` repo. This would enable the job to commit latest generated TPIP report.Same changes I am planning to do for `cpackget`.

Reply:
I tried using both `{{ github.token }}` and `{{ secrets.GITHUB_TOKEN }}`. In both cases GITHUB avoids the further commit actions. 

```
Run EndBug/add-and-commit@1bad3abcf0d6ec49a5857d
Outputs
  committed: false
  commit_long_sha: undefined
  commit_sha: undefined
  pushed: false
  tagged: false
  tag_pushed: false
```

Also, the documentation of the commit action states the same.
https://github.com/EndBug/add-and-commit#the-commit-from-the-action-is-not-triggering-ci
We need to have a PAT with `commit/push` permissions.

Reply:
Thanks for the suggestion. Explicit calling the git push seems working.

Reply:
[Addressed review comments](https://github.com/Open-CMSIS-Pack/cbuild/pull/86/commits/57f7beb7d50ea42ea675a215fb306b49fbf6dc12)